lib/stac-api/index.ts

```diff
@@ -23,12 +24,12 @@ export class PgStacApiLambda extends Construct {
     this.stacApiLambdaFunction = new lambda.Function(this, "lambda", {
       // defaults
       runtime: lambda.Runtime.PYTHON_3_11,
-      handler: "src.handler.handler",
+      handler: "handler.handler",
       memorySize: 8192,
       logRetention: aws_logs.RetentionDays.ONE_WEEK,
       timeout: Duration.seconds(30),
-      code: lambda.Code.fromDockerBuild(__dirname, {
-        file: "runtime/Dockerfile",
+      code: lambda.Code.fromDockerBuild(path.join(__dirname, '..'), {
+        file: "stac-api/runtime/Dockerfile",
```

Comment:
I changed the docker build path so all of these applications could share a single `utils.py` file.